subprojects/docs/src/docs/release/notes.md

```diff
@@ -157,6 +157,7 @@ Starting with Gradle 7.6 this is no longer the case. Gradle recovers from depend
 
 The `dependencies`, `buildEnvironment`, `projects` and `properties` tasks are now compatible with the configuration cache.
 
+The [Maven Publish Plugin](userguide/publishing_maven.html) is now compatible with the configuration cache. Note that when using credentials, the configuration cache requires [safe (empty) credential containers](userguide/configuration_cache.html#config_cache:requirements:safe_credentials).
```

Comment:
```suggestion
The [Maven Publish Plugin](userguide/publishing_maven.html) is now compatible with the configuration cache. 

The configuration cache now requires [safe (empty) credential containers](userguide/configuration_cache.html#config_cache:requirements:safe_credentials).
```

Suggestion: Trimmed a small number of words, and broke this into two separate paragraphs since the credentials change applies to all credentials, not just Maven Publish Plugin credentials (I think?).

Reply:
It is more like an architectural decision, but at this time is only enforced in this plugin.